exercises/chapter3/packages.dhall

```diff
@@ -15,8 +16,8 @@ let overrides =
           , "js-timers"
           , "avar"
           ]
-        , repo = "https://github.com/milesfrain/purescript-test-unit.git"
```

Comment:
We don't need to override that. We just need to update the package set to a newer version:
```console
$ cat packages.dhall 
let upstream =
      https://github.com/purescript/package-sets/releases/download/psc-0.15.2-20220613/packages.dhall
        sha256:99f976d547980055179de2245e428f00212e36acd55d74144eab8ad8bf8570d8

in  upstream
$ spago ls packages | grep 'test'
test-unit                    v17.0.0   Remote "https://github.com/bodil/purescript-test-unit.git"
```
And to clarify, the package set is immutable. So, as long as the package set version isn't updated, that's the same thing as pinning the `test-unit` version to `v17.0.0`.

Reply:
I'm overriding the `test-unit` repo while waiting for this PR to be merged: https://github.com/bodil/purescript-test-unit/pull/47

Reply:
@milesfrain :wave: Hey! How's it been?

Looking at that PR, it's a purely cosmetic change (though it's one I agree with). It's not a blocker for this PR, but it would be a nice change to add to `test-unit`.